### PR TITLE
Add Walling, Tennessee, USA

### LINKED ISSUE #673: Missing City in United States
Missing City
United States (Country) --> Tennessee (State) --> **Walling** (City)

Google Maps : https://goo.gl/maps/4wJf9XhZyxjFQMr48
Wiki: https://en.wikipedia.org/wiki/Walling,_Tennessee

### PR DESCRIPTION
**Pull request by mtsfer**
Reference: https://en.wikipedia.org/wiki/Walling,_Tennessee

Closes #673

### CODE REVIEW

**Comment by dr5hn** [comment]
@mtsfer 
First of all, Thanks for taking time to contribute to this repo. I really appreciate your contribution. 🎉 

The id should be 920 instead of 720

**Comment by dr5hn** [comment]
@mtsfer Please rebase the branch as id now should be 921

**Comment by dr5hn** [comment]
@mtsfer I can still see the conflict. Can you please rebase it again?

**Comment by mtsfer** [comment]
Sorry! I think now its okay.

**Comment by dr5hn** [comment]
@mtsfer Please resolve conflicts.
Screenshot 2024-10-05 at 7 09 31 PM

**Comment by mtsfer** [comment]
Sorry! What about now?

File: sql/world.sql [hunk 1]
```diff
@@ -151285,8 +151285,9 @@ INSERT INTO `cities` (`id`, `name`, `state_id`, `state_code`, `country_id`, `cou
 (153918, 'Hermitage', 3263, 'PW', 140, 'MU', -20.27685080, 57.53471370, '2019-10-05 23:08:25', '2024-09-07 19:08:44', 1, 'Q246286'),
 (153919, 'Marshall', 1400, 'AK', 233, 'US', 61.87811100, 162.08500000, '2024-09-07 18:24:01', '2024-09-07 18:24:01', 1, 'Q79550'),
 (153920, 'Berlin', 1435, 'CT', 233, 'US', 41.622234, -72.773628, '2024-09-27 15:00:00', '2024-09-27 15:00:00', 1, 'Q821199'),
-(153921, 'Stratford', 4069, 'TKI', 158, 'NZ', -39.32215170, 174.26298160, '2024-10-05 16:08:24', '2024-10-05 16:08:24', 1, 'Q1972491'),
+(153921, 'Stratford', 4069, 'TKI', 158, 'NZ', -39.32215170, 174.26298160, '2024-10-05 16:08:24', '2024-10-05 16:08:24', 1, 'Q1972491');
```
Comment (dr5hn) [inline sql/world.sql:151288]: should be `,` at the end.

**Comment by dr5hn** [comment]
@mtsfer 

commit is messed up!

Screenshot 2024-10-05 at 7 30 31 PM

**Comment by mtsfer** [comment]
Nah, let's me close this and open a new one :smiling_face_with_tear:. Zed smh deleted everything

**Pull request closed without merging**